tf2_eigen/CMakeLists.txt

```diff
@@ -1,29 +1,19 @@
 cmake_minimum_required(VERSION 3.5)
 project(tf2_eigen)
 
-find_package(catkin REQUIRED COMPONENTS
-  cmake_modules
-  geometry_msgs
-  tf2
-)
+find_package(ament_cmake REQUIRED)
 
-find_package(Eigen REQUIRED)
+find_package(geometry_msgs REQUIRED)
+find_package(tf2 REQUIRED)
+
+find_package(Eigen3 REQUIRED)
 include_directories(${EIGEN_INCLUDE_DIRS})
 add_definitions(${EIGEN_DEFINITIONS})
```

Comment:
Does this compile on Windows?
@nuclearsandwich might be able to adjust the choco package in case.